docs/docs/graphql.md

```diff
@@ -1253,6 +1253,433 @@ export const getWeather = async ({ input }: WeatherInput) {
 }
 ```
 
+## Self-Documenting GraphQL API
+
+RedwoodJS helps you document your GraphQL API by auto-generating commented SDL that will be incorporated into the `graphql.schema` file when generating types and that will be used for GraphiQL and the GraphQL Playground explorer.
+
+If you comment your Prisma schema models, its fields, or enums, the SDL generator will use those comments as the documentation.
+
+If there is no Prisma comment, then the SDL generator will default a comment that you can then edit.
+
+:::note
+If you re-generate the SDL, any custom comments will be overwritten.
+However, if you make those edits in your Prisma schema, then those will be used.
+:::
+
+### Prisma Schema Comments
+
+Your Prisma schema is documented with a comment `///` that precedes:
+
+* the Model name
+* the Enum name
+* each Model field name
+
+```
+/// A blog post.
+model Post {
+  /// The unique identifier of a post.
+  id        Int      @id @default(autoincrement())
+  /// The title of a post.
+  title     String
+  /// The content of a post.
+  body      String
+  /// When the post was created.
+  createdAt DateTime @default(now())
+}
+
+model Contact {
+  /// The unique identifier of a contact.
+  id        Int @id @default(autoincrement())
+  /// The name of the contact.
+  name      String
+  /// The email of the contact.
+  email     String
+  /// The message of the contact.
+  message   String
+  /// When the contact was created.
+  createdAt DateTime @default(now())
+}
+
+/// A list of allowed colors.
+enum Color {
+  RED
+  GREEN
+  BLUE
+}
+```
+
+### SDL Comments
+
+When used with `--docs` option, [SDL generator](./cli-commands.md#generate-sdl) adds comments for:
+
+* Directives
+* Queries
+* Mutations
+* Input Types
+
+:::note
+By default, the `--docs` option to the SDL generator is false and comments are not created.
+:::
+
+Comments in GraphQL schema are enclosed in `"""` and will be included in the generate4d `.redwood/schema.graphql` at the root of your project.
```

Comment:
I'm not sure I fully understand this comment. But let me try to write what I think it means, and if I'm right, maybe we can see if we should update it somehow

Comments enclosed in `"""` or `"` in your sdl files will be included in the generated GraphQL schema at the root of your project (`.redwood/schema.graphql`).

Does that sound about right?